internal/uvm/cpugroups_test.go

```diff
@@ -0,0 +1,36 @@
+package uvm
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+)
+
+// Unit tests for cpugroup creation, modification, and deletion
+func DisabledTestCPUGroupCreateWithIDAndDelete(t *testing.T) {
```

Comment:
Is this test disabled for some reason?

Reply:
Mentioned in the PR description :)

Reply:
I'd say let's take `Disabled` out of the name, and put a `t.Skip("only works on classic/core scheduler, skipping as we can't check this dynamically right now")` at the top of the test.